core/frontend/src/main/scala/io/udash/bindings/modifiers/DOMManipulator.scala

```diff
@@ -0,0 +1,39 @@
+package io.udash.bindings.modifiers
+
+import org.scalajs.dom.{Element, Node}
+
+private[bindings]
+trait DOMManipulator {
```

Comment:
What's this line break for?

Reply:
Consistency with the other classes in this package.